src/encoded/static/components/browse/components/SearchResultTable.js

```diff
@@ -514,34 +518,34 @@ class ShadowBorderLayer extends React.Component {
             className += ' faded-out';
         }
         return (
-            <div className={className} onMouseDown={this.handleScrollButtonClick.bind(this, 'right')} onMouseUp={this.handleScrollButtonUp} onMouseOut={this.handleScrollButtonUp}>
+            <div className={className} onMouseDown={this.handleRightScrollButtonMouseDown} onMouseUp={this.handleScrollButtonUp} onMouseOut={this.handleScrollButtonUp}>
                 <i className="icon icon-caret-right"/>
             </div>
         );
     }
 
-    performScrollAction(direction = "right", depth = 0){
-        vizUtil.requestAnimationFrame(()=>{
-            var change = (direction === 'right' ? 1 : -1) * this.props.horizontalScrollRateOnEdgeButton;
-            var maxScrollLeft = this.props.fullRowWidth - this.props.tableContainerWidth;
-            var leftOffset = this.props.innerContainerElem.scrollLeft = Math.max(0, Math.min(maxScrollLeft, this.props.innerContainerElem.scrollLeft + change));
-            var detailPanes = DimensioningContainer.findDetailPaneElements();
```

Comment:
Not only is manually finding these detail panes from the DOM here not performant (esp. when done as part of a non-throttled event callback), but the `setDetailPanesLeftOffset` function on next line had also used its own `requestAnimationFrame` call which thus was adding the delay.

`setContainerScrollLeft` now updates the `state,tableContainerScrollLeft` which is held higher in state and which also gets passed down to detailPanes' left offsets.